blueprint/abac/client-devices.md

```diff
@@ -0,0 +1,8011 @@
+
+Microsoft have developed a number of base policies which provide the whitelisting rules for a standard Windows 10 image. These base policies are located on any Windows 10 machine under `%windir%\schemas\CodeIntegrity\ExamplePolicies`. Within that directory, the policy titled `DefaultWindows_Audit.xml` should be leveraged as the base policy. It includes the rules necessary to ensure that Windows, 3rd party hardware and software kernel drivers, and Windows Store apps will run.  
+
+The procedure to leverage this policy is as follows:
+1. Make a copy of the policy and place it into a working directory (i.e. My documents)
+2. Open the policy in a XML editor such as Visual Studio code or notepad.
+3. Remove all flighting related signers (note they will also need to be removed from the signing scenarios)
+4. Using an administrative PowerShell session run
+```powershell
+$WDACPolicy = {path to policy file}
+Set-RuleOption -FilePath $WDACPolicy -Option 0 # Enabled UMCI
+Set-RuleOption -FilePath $WDACPolicy -Option 1 # Enable Boot Menu Protection
+Set-RuleOption -FilePath $WDACPolicy -Option 2 # Require WHQL
+Set-RuleOption -FilePath $WDACPolicy -Option 3 # Enable Audit Mode
+Set-RuleOption -FilePath $WDACPolicy -Option 4 # Disable Flight Signing
+Set-RuleOption -FilePath $WDACPolicy -Option 6 # Enable Unsigned Policy
+Set-RuleOption -FilePath $WDACPolicy -Option 8 # Require EV Signers
+Set-RuleOption -FilePath $WDACPolicy -Option 10 # Enable Boot Audit on Failure
+Set-RuleOption -FilePath $WDACPolicy -Option 12 # Enable Enforce Store Apps
+Set-RuleOption -FilePath $WDACPolicy -Option 13 # Enable Managed Installer
+Set-RuleOption -FilePath $WDACPolicy -Option 16 # Enable No Reboot
+Set-RuleOption -FilePath $WDACPolicy -Option 17 # Enable Allow Supplemental
+Set-RuleOption -FilePath $WDACPolicy -Option 19 # Enable Dynamic Code Security
+```
+5. Deploy the file locally in audit mode and validate no additional files require whitelisting
+6. Using an administrative PowerShell session run the following to switch the policy into enforce mode
+```powershell
+Set-RuleOption -FilePath $WDACPolicy -Option 3 -Delete
+```
+7. Deploy the file locally in enforce mode and validate no additional files require whitelisting
+8. [Deploy the file globally](/blueprint/abac/client-devices.html#wdac-policy---deployment) 
+
+
+### WDAC Policy - Supplementary Policy
+
+Supplementary policies expand on the base policy and allow for whitelisting to be targeted to users and/or groups. Supplementary pollcies can contain both allow and deny rules. A supplementary policy is a base policy until it is linked as a supplementary policy to another base policy. 
+
+Supplementary policies will not work on machines pre 1903. If you are deploying to pre 1903 machines they must be merged into the base policy. [Merge policy](/blueprint/abac/hybrd-client-devices.html#wdac-policy---whitelisting-an-application) instructions are available in the hybrid client whitelisting section.
```

Comment:
Broken link. Was this meant to be linking from cloud-native to hybrid?

Reply:
yes it is as the hybrid talks about merging. Cloud native explains multi policies.